### ISSUE TITLE
Outra vez fallan os proxectos????

### ISSUE THREAD

**Issue by daavoo** [VigoTech/vigotech.github.io#52]
![captura de pantalla de 2017-06-22 14-01-54](https://user-images.githubusercontent.com/12677733/27433102-73266306-5753-11e7-8aba-d8dc738c97e4.png)

![captura de pantalla de 2017-06-22 14-03-03](https://user-images.githubusercontent.com/12677733/27433117-8b454560-5753-11e7-836c-98c1e0c2079a.png)

**Comment by antonmry** [comment]
Sí, a mín tamén. Se teño un rato volvo a desactivalos mentras non demos ca solución definitiva

**Comment by antonmry** [comment]
Desactivados na páxina principal mentras non se arranxe.
